Makefile.darwin

```diff
@@ -21,8 +21,14 @@ FINCH_IMAGE_LOCATION ?= $(OS_OUTDIR)/$(FINCH_OS_BASENAME)
 FINCH_IMAGE_DIGEST ?= "sha512:$(FINCH_OS_DIGEST)"
 
 include $(CURDIR)/deps/container-runtime-full-archive.conf
+ifneq ($(AARCH64_ARTIFACT_PATHING),)
```

Comment:
Why do we need this conditional?

Reply:
Currently we can consume archives directly from upstream nerdctl GitHub releases or from deps.runfinch.com. When pulling from deps.runfinch.com the pathing is different depending on the architecture, so conditional needs to switch based on that.

Reply:
Is it possible that AARCH64_ARTIFACT_PATHING will ever be empty since we also ship the updated config file.

Reply:
Yes, the pathing variable is empty when we consume upstream archives. Makefile will see path is empty and not use that in the location replacement in lima yaml.

i.e.
```
ARTIFACT_BASE_URL=https://github.com/containerd/nerdctl/releases/download/v1.7.7

AARCH64_ARTIFACT_PATHING=
AARCH64_ARTIFACT=nerdctl-full-1.7.7-linux-arm64.tar.gz
AARCH64_256_DIGEST=b161a20c0e41f9ad999e8411e23c58ece4b3e584ae90b4252b76a39eee4a0c31
```